src/winter_onboarding_2021/fleet_management_service/models/fleet.clj

```diff
@@ -2,10 +2,21 @@
   (:require [winter-onboarding-2021.fleet-management-service.db.fleet :as fleet-db]
             [winter-onboarding-2021.fleet-management-service.utils :as utils]
             [winter-onboarding-2021.fleet-management-service.specs :as specs]
-            [winter-onboarding-2021.fleet-management-service.error :as error]))
+            [winter-onboarding-2021.fleet-management-service.error :as error]
+            [winter-onboarding-2021.fleet-management-service.config :as config]))
 
 (defn create [fleet]
   (let [fleet-with-valid-keys (utils/select-keys-from-spec fleet ::specs/fleets)]
     (if (empty? fleet-with-valid-keys)
       error/no-valid-keys
       (fleet-db/create fleet-with-valid-keys))))
+
+(defn total []
+  (fleet-db/total))
+
+(defn select-fleets-for-admin
```

Comment:
Ditto what Govind said about the fn with the same name in the db layer above. Consider `user-fleets`